test/css_url_to_erb_asset_path_test.rb

```diff
@@ -14,6 +14,17 @@ def test_no_url
     def test_url_with_data
       css = "background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaskip);"
       assert_equal css, filter.call(css, "/current/file")
+      css = "background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaskip');"
+      assert_equal css, filter.call(css, "/current/file")
```

Comment:
Oh! I didn't notice quotes.
Do you mind to split this into separate commit then? Because it is not related to your "http://" problem.